install.sh

```diff
@@ -120,6 +123,63 @@ echo "When output says DB is ready to receive connections, hit CTRL+C and move o
 
 sleep 5 #So that user reads the above lines properly
 
-docker-compose up
+gnome-terminal -- docker-compose up
```

Comment:
Do not expect that people will be using gnome-terminal. I use Konsole myself.

Reply:
Can you try with `docker-compose up -d`? Then I guess you'll not have to rely on a separate window in the terminal.